### ISSUE TITLE
Config Flow Not Connecting

### ISSUE THREAD

**Issue by kernelpanic85** [iloveicedgreentea/jvc_homeassistant#40]
I recently upgraded from 3.7.3 to 4.1.1 and with that HA suggested I move to the config flow instead of YAML. I moved the config over and since then, HA is not connecting to my projector and I'm getting the following repeatedly in the logs:

Logger: homeassistant.helpers.entity
Source: helpers/entity.py:898
First occurred: February 18, 2024 at 4:53:21 PM (289 occurrences)
Last logged: February 18, 2024 at 11:06:01 PM

Update for remote.theater_projector fails
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 898, in async_update_ha_state
    await self.async_device_update()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 1216, in async_device_update
    await self.async_update()
  File "/config/custom_components/jvc_projectors/remote.py", line 354, in async_update
    attribute_getters.append(
TypeError: list.append() takes exactly one argument (2 given)

PS. To mitigate confusion, it might be helpful if the first and last fields in the config flow dialog were labelled so the user can clearly understand what information each box is looking for:

![image](https://github.com/iloveicedgreentea/jvc_homeassistant/assets/80057019/2e44203d-8a42-41b1-ba32-046865eea3b3)

**Comment by iloveicedgreentea** [comment]
Which model do you have? Can you turn on debug logs and send logs

```logger:
  default: warning
  logs:
    homeassistant.config_entries: debug
    custom_components.jvc_projectors: debug```

**Comment by iloveicedgreentea** [comment]
nvm I found the error. You must have a lamp model which I didn't catch because I can't test those commands. Will be fixed shortly. I will add fields for those too

**Comment by kernelpanic85** [comment]
Thanks! Yes, I have the NP5 running 3.0.

**Comment by iloveicedgreentea** [comment]
Please try 4.2.0

**Comment by kernelpanic85** [comment]
Unfortunately, it still doesn't seem to be working. I ran the debug logs and this is all I see:

```
2024-02-20 12:33:50.195 DEBUG (MainThread) [custom_components.jvc_projectors.remote] Setting up JVC Projector with options: JVCInput(host='192.168.2.73', password='<removed>', port=20554, connect_timeout=3)
2024-02-20 12:33:50.207 DEBUG (MainThread) [custom_components.jvc_projectors.remote] Projector was not previously connected, skipping reconnection.
```

**Comment by iloveicedgreentea** [comment]
Did you try to turn it on?

**Comment by kernelpanic85** [comment]
I had it on, but not via HA. I tried this time from HA and got a bunch of logs. It actually locked up the projector where it wasn't responding to the remote and I had to reboot it.

Here are the logs:
[projector debug.txt](https://github.com/iloveicedgreentea/jvc_homeassistant/files/14353342/projector.debug.txt)

**Comment by iloveicedgreentea** [comment]
Can you try 4.2.1? I fixed a few more potential issues

**Comment by kernelpanic85** [comment]
Yes, all good now! Thanks!

**Comment by kernelpanic85** [comment]
Quick follow up on this, everything is working fine except that when the projector turns off via the remote, HA does not pick that up and still shows it on hours later and I had to toggle it off in HA manually. Below is what I see in the debug logs starting from when I turned the projector off via the remote (around 14:02) and then ending with toggling manually in HA (around 17:21).

[home-assistant_jvc_projectors_2024-02-26T22-25-16.654Z.log](https://github.com/iloveicedgreentea/jvc_homeassistant/files/14412119/home-assistant_jvc_projectors_2024-02-26T22-25-16.654Z.log)

I can open a separate issue if you like.
